bootstrap/kubeadm/config/rbac/service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: manager-cabpk
```

Comment:
Could we leverage the name prefix automation here? Seems we could make use of `namePrefix` transformer in multiple places instead of adding these suffix directly

/hold